player/lua/select.lua

```diff
@@ -326,6 +295,46 @@ mp.add_key_binding(nil, "select-subtitle-line", function ()
             mp.commandv("seek", sub_times[index] + delay, "absolute")
         end,
     })
+end
+
+mp.add_key_binding(nil, "select-subtitle-line", function ()
+    local sub = mp.get_property_native("current-tracks/sub")
+
+    if sub == nil then
+        show_warning("No subtitle is loaded.")
+        return
+    end
+
+    if sub.external and sub["external-filename"]:find("^edl://") then
+        sub["external-filename"] = sub["external-filename"]:match('https?://.*')
+                                   or sub["external-filename"]
+    end
+
+    local screenx, screeny = mp.get_osd_size()
+    mp.osd_message("Subtitle export/conversion in progress.", 3)
+    mp.set_osd_ass(screenx, screeny, "{\\an9}● ")
+    mp.command_native_async({
+        name = "subprocess",
+        capture_stdout = true,
+        args = sub.external
+            and {"ffmpeg", "-loglevel", "error", "-i", sub["external-filename"],
+                 "-f", "lrc", "-map_metadata", "-1", "-fflags", "+bitexact", "-"}
+            or {"ffmpeg", "-loglevel", "error", "-i", mp.get_property("path"),
+                "-map", "s:" .. sub["id"] - 1, "-f", "lrc", "-map_metadata",
+                "-1", "-fflags", "+bitexact", "-"}
+    }, function(success, result)
```

Comment:
Can you use `function (` for consistency with the rest?

Reply:
Done.